code/modules/reagents/reagent_containers/food/snacks/meat.dm

```diff
@@ -36,6 +38,8 @@
 	bitesize = 6
 	preloaded_reagents = list("protein" = 4, "blattedin" = 8, "diplopterum" = 7)
 
+	price_tag = 200
```

Comment:
Hmm, those prices looks expensive. Isn't 200 credits for a piece of roach meat super expensive when good old corgi meat is 25 and cutlet 100.